attestation-service/README.md

```diff
@@ -40,13 +40,22 @@ from the Attestation Agent inside the guest.
 
 The AS has a two-step verification process.
 
-1. Verify the format and provenance of evidence itself (i.e. check the signature of the evidence).
-2. Evaluate the claims presented by the evidence (i.e. check that measurements are what the client expects).
+1. Verify the format and provenance of evidence itself (i.e. check the signature of the evidence). This is done by [verifier drivers](#verifier-drivers).
+2. Evaluate the claims presented by the evidence (i.e. check that measurements are what the client expects). This is done by [policy engine](#policy-engine) and [RVPS](#reference-value-provider-service).
 
 The first step is accomplished by one of the platform-specific [Verifier Drivers](#verifier-drivers).
 The second step is driven by the [Policy Engine](#policy-engine) with help from the [RVPS](#reference-value-provider-service).
 
-# Usage
+# Quick Start
+
+Please refer to
+- [Restful CoCo AS](docs/restful-as.md#quick-start)
+- [gRPC CoCo AS](docs/grpc-as.md#quick-start)
```

Comment:
Maybe we should also mention somewhere that the AS can be consumed as a crate.

Reply:
Right. Only 4 lines later, in `Advanced Topics` the doc will say it could be used as a library/crate.